src/main/scala/com/codacy/client/stash/service/ProjectServices.scala

```diff
@@ -10,17 +10,23 @@ class ProjectServices(client: StashClient) {
   /**
     * Only projects for which the authenticated user has the PROJECT_VIEW permission will be returned.
     */
-  def findById(projectKey: String): RequestResponse[Project] = {
-    client.execute(Request(s"$BASE/$projectKey", classOf[Project]))()
+  def findById(projectKey: String, includeAvatar: Boolean = false): RequestResponse[Project] = {
+    val params = if (includeAvatar) Map("avatarSize" -> "64") else Map.empty[String, String]
```

Comment:
Should we abstract this? we could create something like a withAvatar where you would pass this boolean and then call a body with the params?